test/functional/api/cas/cli_messages.py

```diff
@@ -83,6 +82,33 @@
     r"No caches running"
 ]
 
+unavailable_device = [
+    r"Error while opening \'\S+\'exclusively\. This can be due to\n"
+    r"cache instance running on this device\. In such case please stop the cache and try again\."
+]
+
+error_handling = [
+    r"Error during options handling"
+]
+
+no_cas_metadata = [
+    r"Device \'\S+\' does not contain OpenCAS's metadata\."
+]
+
+cache_dirty_data = [
+    r"Cache instance contains dirty data\. Clearing metadata will result in loss of dirty data\.\n"
+    r"Please load cache instance and flush dirty data in order to preserve them on the core "
+    r"device\.\n"
+    r"Alternatively, if you wish to clear metadata anyway, please use \'--force\' option\."
+]
+
+cache_dirty_shutdown = [
+    r"Cache instance did not shut down cleanly\. It might contain dirty data\. \n"
+    r"Clearing metadata might result in loss of dirty data\. Please recover cache instance\n"
+    r"by loading it and flush dirty data in order to preserve then on the core device\.\n"
```

Comment:
typo: then -> them

Reply:
nice catch! The typo is present in casadm as well so fix is needed in both places

Reply:
@karolinavelkaja please create a low priority bug for the typo in casadm

Reply:
Test consists fix for typo, buf created and should be linked: https://github.com/Open-CAS/open-cas-linux/issues/917

Reply:
Fix in casadm merged, test is up-to date (ready to review&merge)